setup/gcp/installer/forseti_server_installer.py

```diff
@@ -213,16 +196,15 @@ def get_configuration_values(self):
             'EMAIL_SENDER': self.config.notification_sender_email,
             'SENDGRID_API_KEY': self.config.sendgrid_api_key,
             'SCANNER_BUCKET': bucket_name[len('gs://'):],
-            'DOMAIN_SUPER_ADMIN_EMAIL': self.config.gsuite_superadmin_email,
-            'ENABLE_GROUP_SCANNER': 'true',
+            'DOMAIN_SUPER_ADMIN_EMAIL': self.config.gsuite_superadmin_email
         }
 
-    def determine_access_target(self):
+    def determine_access_level(self):
```

Comment:
I think that "target" as before, makes more sense than "level".  Can we please change it back to `determine_access_target`?